clang-tools-extra/clang-tidy/modernize/MinMaxUseInitializerListCheck.cpp

```diff
@@ -72,7 +72,7 @@ static FindArgsResult findArgs(const CallExpr *Call) {
   return Result;
 }
 
-static SmallVector<FixItHint>
+static std::pair<bool, SmallVector<FixItHint>>
```

Comment:
Could you add some comments of this result. since bool means nothing.

Reply:
done